python/pyspark/sql/dataframe.py

```diff
@@ -897,19 +898,31 @@ def show(self, n: int = 20, truncate: Union[bool, int] = True, vertical: bool =
         Parameters
         ----------
         n : int, optional
-            Number of rows to show.
+            Number of rows to show. Default is 20.
```

Comment:
I checked all the other cases.
Shall we change to `default 20. Number of rows to show.` ?

Reply:
Hmm I didn't get the point.  Do you mean changing `Default is 20` to `Default: 20 rows to show` since the truncation is also 20 characters?

Reply:
`Number of rows to show` to `default 20. Number of rows to show.`